### DISCUSSION TITLE
How to set up custom config file?

### DISCUSSION THREAD

**Discussion by emmabubu** [Synthetixio/synpress#593] [Q&A]
Hi Guys,

Does anyone have experience in setting up custom config file in the newest version of Synpress(3.0.5)?

I tried copy the default synpress config file to project root folder, changed the base url, then run `synpress run --spec tests/e2e/specs/mySpecFile ---configFile ./customConfig.config.js`, it gave me the error below:
<img width="1349" alt="Screen Shot 2022-12-06 at 16 46 15" src="https://user-images.githubusercontent.com/100118606/205863475-6b2a76c0-7d5b-48de-b473-db599cc6d263.png">

Is there anything else that has to be modified in copied default config file to use it as custom config file?

**Comment by mitsudani** [comment]
I'm having the exact same problem. I also tried to use a smaller config file, just changing the baseUrl like other users did but I got the same error.
